.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.4.1
+addons:
+  chrome: stable
+before_script:
+  - bundle install
+  - bundle exec rails db:create db:test:prepare
+script:
+  - bundle exec rake
+install:
+  - wget -N http://chromedriver.storage.googleapis.com/2.33/chromedriver_linux64.zip -P ~/
```

Comment:
I don’t understand. Why are you installing Chrome on Travis? You are not using Chromedriver in your tests.